Example/Example/Common/BankCard/BankCardViewController.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  BankCardViewController.swift
+//  Example
+//
+//  Created by Artem Starosvetskiy on 15/12/2016.
+//  Copyright © 2016 Artem Starosvetskiy. All rights reserved.
+//
+
+#if os(macOS)
+    import Cocoa
+#else
+    import UIKit
+#endif
+
+import TextInputKit
+
+final class BankCardViewController : ViewController {
```

Comment:
I would recommend create two separate controllers for mac OS and iOS.
Mixing UI/NS Text Views may be confusing due to different set of methods.